### PR TITLE
Bump version to 4.0.0

### PR DESCRIPTION
**Pull request by mdo**
It's about time—we're shipping this thing! This bumps the version from `4.0.0-beta.3` to `4.0.0`.

/cc @twbs/team

### CODE REVIEW

**Review by mattonit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -38,11 +38,11 @@
 
 Several quick start options are available:
 
-- [Download the latest release.](https://github.com/twbs/bootstrap/archive/v4.0.0-beta.3.zip)
+- [Download the latest release.](https://github.com/twbs/bootstrap/archive/v4.0.0.zip)
 - Clone the repo: `git clone https://github.com/twbs/bootstrap.git`
-- Install with [npm](https://www.npmjs.com/): `npm install bootstrap@4.0.0-beta.3`
-- Install with [yarn](https://yarnpkg.com/): `yarn add bootstrap@4.0.0-beta.3`
-- Install with [Composer](https://getcomposer.org/): `composer require twbs/bootstrap:4.0.0-beta.3`
+- Install with [npm](https://www.npmjs.com/): `npm install bootstrap@4.0.0`
```
Comment (Johann-S) [inline README.md]: `npm install bootstrap` should be enough ?

Comment (mdo) [reply README.md]: Yeah, we can clean that up afterwards. Let's roll with it like we've done the last couple releases :).

Comment (mdo) [reply README.md]: Nevermind, see our Slack! Updating this and the download page.

**Comment by sagudev** [comment]
One check failed and these changes were made on old dev branch

**Pull request merged**